quirks/password-rules.json

```diff
@@ -120,6 +120,9 @@
     "girlscouts.org": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$#!];"
     },
+    "github.com": {
+        "password-rules": "minlength: 8; required: lower; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789`!@#$%^&*()+~{}'\";:<>?];"
```

Comment:
This is equivalent to the easier to read: `minlength: 8; required: lower; required: digit; allowed: lower, upper, digit, [`!@#$%^&*()+~{}'\";:<>?];`

(I should fix the girlscouts.org rule, which is currently setting a bad example.)

Reply:
@rmondello -- Thanks for the suggestion. I fixed it (along with the girlscouts.org rule as well)